app/db/migrations/versions/1787d27cfda_.py

```diff
@@ -0,0 +1,33 @@
+# flake8: noqa
```

Comment:
I didn't do this intentionally, it was a byproduct of the way I wrote the work. First I created the basic blueprint layout, then I added the DB, then I created the Identity class. Later on I added the different pieces of the Identity when I needed them. So the multiple migrations are a product of working to a goal (first "Store Identities in a DB and list them", second "Store the info needed by Verify in Identities").

In terms of whats better, meh. Shouldn't really matter either way. If we are worried about migrations not applying then we'd have bigger problems.